protocol/protobuf/chat_message.pb.go

```diff
@@ -1,24 +1,24 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
```

Comment:
Could you please configure your env to have the same configurations as other devs have in order not to generate proto which are different due to different set-up?

1. removed all my manual go-related installations (like protobuf and etc)
2. run make setup , make setup-dev
3. It will generate ~/go folder. Add path ~/go/bin to PATH
4. Done, version should be the correct

Reply:
we use `shell.nix`, I think that's the best way to replicate dev environments, maybe @jakubgs can give his opinion. I can try to make it the same as what we normally use, but I would lean towards all of us using `shell.nix` rather than depending on the `os` packages, since people might have different package managers etc

Reply:
I think resurrecting https://github.com/status-im/status-go/issues/4191 would save us from such discussions.

Reply:
I tried but the env doesn't install on my system, (shell.nix works, but the version specified is too old), is it ok if I merge as it is?

Reply:
@apentori is already working on this in:

* https://github.com/status-im/status-go/pull/4388

Just other priorities took his focus away from it.

Reply:
@jakubgs thanks for update :pray: